app/src/test/kotlin/com/x8bit/bitwarden/ui/platform/feature/settings/accountsecurity/pendingrequests/PendingRequestsViewModelTest.kt

```diff
@@ -139,7 +141,11 @@ class PendingRequestsViewModelTest : BaseViewModelTest() {
                     PendingRequestsState.ViewState.Content.PendingLoginRequest(
                         fingerprintPhrase = "pantry-overdue-survive-sleep-jab",
                         platform = "Android",
-                        timestamp = nowZonedDateTime.format(dateTimeFormatter),
+                        timestamp = nowZonedDateTime.toFormattedDateTimeStyle(
```

Comment:
Using `toFormattedDateTimeStylel` fixes tests failures where the system injects `NBBSP` instead of a standard space in the formatted string.